src/realm/object-store/util/bson/bson.cpp

```diff
@@ -484,6 +492,12 @@ std::ostream& operator<<(std::ostream& out, const Bson& b)
             out << "]";
             break;
         }
+        case Bson::Type::Uuid: {
```

Comment:
UUID doesn't have it's own BSON designation unfortunately, it is subtype 4 of binary; see https://github.com/realm/realm-core/blob/master/src/realm/obj.cpp#L904-L907 for example. I know this is only a draft PR at this point so I may be jumping the gun here, but please add some tests for the new code in tests/object-store/bson.cpp

Reply:
Thanks @ironage . I'll add the tests here. The code by this link looks faulty with "}}" instead of "}"